apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console/GoogleCloudConsoleFormatterOptions.cs

```diff
@@ -32,4 +31,13 @@ public class GoogleCloudConsoleFormatterOptions : ConsoleFormatterOptions
 
     // That said, for actual Google Cloud Logging, very few things are really flexible -
     // the property names for state and scopes are the only things that come to mind.
+
+    /// <summary>
+    /// The ID of the Google Cloud Project where trace data is being written to Google Cloud Trace.
+    /// This property is optional and should be specified if you want to navigate from a log entry to a trace.
```

Comment:
This sounds like it's purely for navigation - it won't affect whether traces are actually being recorded in Cloud Trace. Is that the case, or does enabling this mean you get tracing (via log entries) with no other work? I haven't been following the twists and turns of what actually creates the trace.